src/messages.ts

```diff
@@ -0,0 +1,12 @@
+const messages = {
```

Comment:
instead of defining the translations in here i think it might be better to pass in a message formatter function in the highlighter options so we can do all this in app land